### ISSUE TITLE
Parsing the implied meaning of an Interrogative Word

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#41]
There should probably be further language on interpretation of the interrogative word. Using different interrogative words CAN result in the answer content being different. The use of some interrogative words could be considered incorrect. 

But all of that should be fleshed out some. 

This applies mainly to quizzer provided reference questions.

The specific trailing interrogative word rarely makes a difference to the quizzer on INT questions. 
It only makes a difference if it changes the answer text.

**Comment by jswingle** [comment]
I agree, but I think it would be super easy to accidentally do this in a way that makes things _worse_, so a good deal of thought and discussion would need to go into it.

The thing that immediately jumps to my mind would be clarifying whether or not a "when" interrogative can refer to an "if" clause, e.g. "You are illegitimate children when?" A: "If you are not disciplined." This is a frequent point of contention, and would be easy to write a rule clarifying one way or the other.

**Comment by scottpeterson** [comment]
I have limited time as well! hah. 

I'd love if others can help clarify my own thoughts. 

Generally, we don't want the "question-writer-chosen interrogative word" to be part of the competition.  BUT, it absolutely plays a role, even on INTs and MAs. If the question writer writes "Where created the heavens and the earth?" with the answer "God", that's very confusing to the quizzer.  The "confusing and misleading" clause of Invalid Questions is what we use to make these sorts of chosen interrogative words invalid. 

When it comes to a quizzer providing an interrogative word on a reference question, we really need to think similarly. A quizzer shouldn't be required to provide a question with the same interrogative word as the written question.  But the quizzer's chosen interrogative word should still be held to the same not "confusing or misleading" clause. 

I do think that is generally what is done. Quizmasters try to find ways to "allow" whatever interrogative word the quizzer uses when providing a reference question.

I guess I want an additional bit of wording for "valid questions" that is something like: "The chosen interrogative word (chosen either by the question writer, or by a quizzer providing a reference question) should require the written answer text.   If the chosen interrogative word requires a different amount of answer text, it will be considered an invalid question."

**Comment by jswingle** [comment]
Not to impose too much time on you! But there's so many issues to work through that it's: "well OK this sounds like an intriguing idea but we need to start forcing some progression on it if it's going to be an option in the 2023-2024 rulebook" and the sensible  first person to turn to is whoever opened the issue.

I agree with your thoughts. My general thoughts about the quizzing environment we want:

- As much as possible we want to reward material knowledge, _not_ the ability to form a question and understand the rules of quizzing
- Requiring the quizzer to give the question on a CR/CVR will, _to some extent or another_, conflict with the above desire, and that's fine, but it should be balanced carefully
- Therefore if I can in any way construe the quizzer's given question as "valid," I'm going to count them right, even if it's a hot garbage question that I would fire an actual question writer for writing
- To maintain consistency, though, this means that I as a QM or AJ cannot begin throwing out questions which appear during a quiz simply because I "don't like them" - they would have to be objectively "invalid"

**Comment by gryphonshafer** [comment]
> As much as possible we want to reward material knowledge, not the ability to form a question and understand the rules of quizzing

This. So much this. I regret that I can only agree with the above 100%.

**Comment by jswingle** [comment]
The difficult part about writing a rule concerning this, is that we're sort of just saying "the English language matters." You cannot ask "What went up to Jerusalem?" with the answer "Jesus" because that's universally and objectively considered to be a confusing, misleading, and wrong use of the English language. So for that reason, I wouldn't allow a quizzer to give it as a question. They're wrong, even if the question "What went up to Jerusalem?" has no alternative answer in the text.

But if something is at all ambiguous or debatable, I'm inclined to give it to the quizzer. Remember the controversial ruling at GWI a few years back? The interrogative word was "Where" and the answer was "from heaven" or something like that. @scottpeterson was QM, the table ruled against the quizzer but I would have ruled in their favor because I thought it was unreasonable to pick apart the grammar that much. But I definitely _understood_ why the table ruled the way it did. It's a tough issue.

**Comment by gryphonshafer** [comment]
Hypothetically, consider quizzers for which English is their second language? They memorize and recall/recite the material word-perfect. And then provide a "grammatically-bonkers" question.

This is anti-mission.

**Comment by jswingle** [comment]
Ehh, I get what you're saying, but unless we change every single question to a FTV or Quote, we're not going to totally avoid being less accessible to ESL quizzers. At the end of the day we have to use interrogative words and we're going to have to use them in ways that are sensible in the lingua franca.

**Comment by jswingle** [comment]
I don't want officials to be in the business of nitpicking a quizzer's given question to see whether or not it's technically grammatically valid. I do think officials are more or less forced to require quizzers to give a question that makes _some sort of sense_ of the material, though. You can't use the interrogative "where" to refer to the phrase "during the Passover" or the interrogative "why" to refer to the phrase "Jesus our Lord"

**Comment by scottpeterson** [comment]
yep, we tried and failed to find a way grammatically for the chosen interrogative word "where" to be answered by "from heaven."

**Comment by kclimenhaga** [comment]
Part of the problem is that English uses a lot of question word + preposition combinations. The correct question word for "from heaven" would be "from where", which we obviously don't allow. We could go Shakespearean and allow "Whence", which would work better than "how", which is what I assume was on the card. However, I suspect that most people would oppose that (for good reason). 

I am in favour of allowing even technically ungrammatical questions in fuzzy cases like this. Modern English does not have a question word to provide the answer "from heaven". As a question writer, I would use "how", but that doesn't really make sense either (How did you come to the quiz meet? From Canada.) I'm not sure how to translate this into a rule. The only concrete rules I use are 1. Who/whom are interchangeable for quizzers, 2. Quizzers can use who/what for fuzzy cases (The bread of life = I am who or I am what), 3. If the question word given would require a different answer, and there is an unambiguously correct question word, the quizzer is incorrect.

**Comment by jswingle** [comment]
As an aside, this sort of goes into my pet peeve about "the Spirit" needing to be the answer to a "who" question and "the spirit" needing to be the answer to a "what" question.

I am consciously inconsistent about this when I quizmaster. I think questions are objectively invalid and need to be thrown out if they use "what" to refer to the Spirit, but if a quizzer's given question on a CR/CVR uses a "what" I will count them right because I think it's unreasonable to require quizzers to know which uses of "spirit" the NIV translators are interpreting to be a reference to the Holy Spirit. (And -- we don't really want to be testing quizzers on their theological knowledge, even though the personhood of the Spirit is super basic and is already part of the rules in the Deity/Trinity Rule.) 

My intuition on this is to be inconsistent because it just seems like the fairest way to do it.
